pkgs/by-name/li/libnvidia-container/package.nix

```diff
@@ -85,11 +96,25 @@ stdenv.mkDerivation rec {
   '';
 
   env.NIX_CFLAGS_COMPILE = toString [ "-I${libtirpc.dev}/include/tirpc" ];
-  NIX_LDFLAGS = [ "-L${libtirpc.dev}/lib" "-ltirpc" ];
+  NIX_LDFLAGS = [
+    "-L${libtirpc}/lib"
```

Comment:
(not a blocker) While at it: there are `get{Lib,Dev,Include}` functions which offer fallback branches and allow changing the target package's output sets